@emeryberger The issue raised in the validation is: 

Processing Xiaoyu Xia,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/x/xia-dr-xiaoyu,v-6hODoAAAAJ
  Checking https://dblp.org/search/author/api?q=author%3AXiaoyu%20Xia:$%3A&format=json&c=10
  WARNING: Possibly invalid name (Xiaoyu Xia). This may be a disambiguation entry.
  Checking homepage URL (https://www.rmit.edu.au/contact/staff-contacts/academic-staff/x/xia-dr-xiaoyu)
SUCCESS: Page loaded successfully with requests.
***Test failure for Xiaoyu Xia***.
Error: Process completed with exit code 255. 

The same information was added in Pull Request #6396 before but not sure why it was removed.

--- a/csrankings-x.csv
+++ b/csrankings-x.csv
@@ -308,7 +308,7 @@ Xiaoyong Du 0001,Renmin University of China,http://iir.ruc.edu.cn/~duyong,NOSCHO
 Xiaoyong Li 0003,BUPT,https://scss.bupt.edu.cn/info/1063/1147.htm,NOSCHOLARPAGE
 Xiaoyu He 0001,Sun Yat-sen University,https://hxyokokok.github.io/,qWyVzoQAAAAJ
 Xiaoyu Sun 0002,Australian National University,https://comp.anu.edu.au/people/xiaoyu-sun,RB1nBOUAAAAJ
-Xiaoyu Xia,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/x/xia-dr-xiaoyu,v-6hODoAAAAJ
+Xiaoyu Xia 0001,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/x/xia-dr-xiaoyu,v-6hODoAAAAJ
 Xiaoyuan Jing,Wuhan University,http://mla.whu.edu.cn,NOSCHOLARPAGE
 Xiaoyuan Xie,Wuhan University,http://xiaoyuanxie.github.io,VlHMPKoAAAAJ
 Xiaoyun Wang 0001,Tsinghua University,http://www.tsinghua.edu.cn/publish/casen/1695/2010/20101224093253705266640/20101224093253705266640_.html,5pUP56sAAAAJ

